cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -64,6 +64,10 @@ def filtered_list(cls):
         if not settings.FEATURES.get('ENABLE_VIDEO_UPLOAD_PIPELINE'):
             filtered_list.append('video_upload_pipeline')
 
+        # Do not show facebook_url if the feature is disabled.
```

Comment:
This `filtered_list` is a list of settings that should be *removed* from the UI.  And we remove them only if its feature flag is *not* enabled.  That's why we check for `if not`. 

Is that what you were asking?